package.json

```diff
@@ -109,7 +109,6 @@
     "request": "^2.88.0",
     "semver": "^6.3.0",
     "shelljs": "^0.8.3",
-    "storybook-chromatic": "^3.0.0",
```

Comment:
try re-adding this

Reply:
Yeah, read too much into it and removed it entirely when it's just talking about the `config.js`. Nice catch.

Reply:
![](https://media.giphy.com/media/QyCBAUlaXdzpu/giphy.gif)